submodules/LIB/build.mk

```diff
@@ -113,6 +113,18 @@ fpga-debug:
 fpga-clean:
 	make -C $(FPGA_DIR) clean
 
+
+
+
+#
+#Caravel
+#
+Caravel_DIR=~/iob_soc/submodules/caravel_user_project 
+caravel-build:
+	make -C $(Caravel_DIR) -j1 build
```

Comment:
this file becomes the Makefile at the root build directory 

CARAVEL must be inside the build directory , not in the repo space 

Use relative paths -- no need for absolute paths 

make -C caravel is enough if CARAVEL_DIR is at the top of the build dir